language/bert_extraction/steal_bert_qa/data_generation/preprocess_thief_dataset_squad.py

```diff
@@ -16,13 +16,13 @@
 import json
 import random
 
-from bert_extraction.steal_bert_qa.data_generation import preprocess_util as pp_util
+from language.bert_extraction.steal_bert_qa.data_generation import preprocess_util as pp_util
```

Comment:
@kentonl could you confirm this? I vaguely remember this was correct in November, but have things changed now?

Reply:
When I did run this without the language module name prepended it ran into errors telling that bert_extraction doest exist etc. Maybe there is a difference in the way we installed the language package? At the top most level of language repo, I build it using the command
`python3 setup.py build && python3 setup.py install`

Reply:
Seems okay, I did find the `from language.` prefix in other places of the repository. @kentonl does this break things when the code is compiled internally?